packages/utils/tokens/src/layout-tokens.ts

```diff
@@ -5,16 +5,27 @@ import { tokenBuilder } from './tokenBuilder';
 
 export interface LayoutTokens {
   minWidth?: ViewStyle['minWidth'];
+  maxWidth?: ViewStyle['maxWidth'];
   minHeight?: ViewStyle['minHeight'];
   padding?: ViewStyle['padding'];
   paddingHorizontal?: ViewStyle['paddingHorizontal'];
+  paddingVertical?: ViewStyle['paddingVertical'];
```

Comment:
nit: This package pulls in from `@uifabricshared/foundation-tokens` on line 2, which I think is deprecated but used in some places? I saw it used in Floating Action Button. Is that something we need to stop referencing or is it OK?

Reply:
Hmmm I'll look into it.

Reply:
Hm yeah I think that'll be its own thing. https://github.com/microsoft/fluentui-react-native/issues/1639